### PR TITLE
Update L10N resource files with 4.6 strings from Transifex (20151129)

### PR DESCRIPTION
**Pull request by milamberspace**
Small update of L10N files before the 4.6.1 release candidate

cc @remibergsma

### CODE REVIEW

**Comment by pdion891** [comment]
LGTM, seams to fix linguistic issues, at least solving few for French.

File: client/WEB-INF/classes/resources/messages_fr_FR.properties [hunk 1]
```diff
@@ -1739,7 +1740,7 @@ message.action.enable.nexusVswitch=Confirmer l\\'activation de ce Nexus 1000v
 message.action.enable.physical.network=Confirmer l\\'activation de ce r\u00e9seau physique.
 message.action.enable.pod=Activer ce pod ?
 message.action.enable.zone=Activer cette zone ?
-message.action.expunge.instance=Confirmez que vous souhaitez oruger cette instance.
+message.action.expunge.instance=Confirmez que vous souhaitez effacer cette instance.
```
Comment (pdube) [inline client/WEB-INF/classes/resources/messages_fr_FR.properties]: purger?

**Comment by pdube** [comment]
Suggested a small french change, but still LGTM

**Comment by remibergsma** [comment]
Thanks for the reviews, I'll merge it to 4.6 like this. The improvement should go via Transiflex website and ends up in a new PR when @milamberspace submits the new translations.

**Comment by pdube** [comment]
@remibergsma Fine with me

**Pull request merged**